src/Raven.Client/ServerWide/Operations/UpdateDocumentsCompressionConfigurationOperation.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Client.Json.Serialization;
+using Raven.Client.Util;
+using Sparrow.Json;
+
+namespace Raven.Client.ServerWide.Operations
+{
+    class UpdateDocumentsCompressionConfigurationOperation : IMaintenanceOperation<DocumentCompressionConfigurationResult>
```

Comment:
This is implicitly set to `internal`, but we want this to be `public` ?

Reply:
Done